core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeSorterSpillReader.java

```diff
@@ -148,4 +141,34 @@ public void close() throws IOException {
      }
    }
   }
+
+  private void readFile() throws IOException {
+    assert (dataFile.length() > 0);
+    final ConfigEntry<Object> bufferSizeConfigEntry =
+            package$.MODULE$.UNSAFE_SORTER_SPILL_READER_BUFFER_SIZE();
```

Comment:
Probably, your copy-and-paste lead to wrong indents in this method. Could you check them again?

Reply:
Yah. My Intellij had 8 spaces tab. I will fix it. Thank you